### ISSUE TITLE
 Broadcast of Intent { act=android.intent.action.SCREEN_OFF flg=0x50200010 (has extras) }

### ISSUE THREAD

**Issue by livezencorp** [googleads/googleads-mobile-unity#973]
Unity version : 2019.1.9f1
Google Mobile Ads Unity Plugin v3.18.1

"main" tid=1 Blocked 
"main" prio=5 tid=1 Blocked
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x74d5ee00 self=0x7e10ac4a00
  | sysTid=9762 nice=0 cgrp=default sched=0/0 handle=0x7e162df9c8
  | state=S schedstat=( 13987713472 3090126685 21001 ) utm=665 stm=733 core=4 HZ=100
  | stack=0x7fe109f000-0x7fe10a1000 stackSize=8MB
  | held mutexes=
  at com.google.android.gms.ads.internal.webview.v.E (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827)
- waiting to lock <0x0388bcd5> (a com.google.android.gms.ads.internal.webview.v) held by thread 5
  at com.google.android.gms.ads.internal.webview.t.E (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:8)
  at com.google.android.gms.ads.internal.webview.j.onPageFinished (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:2)
- locked <0x0c7f7cea> (a java.lang.Object)
  at uU.d (PG:307)
  at aIV.handleMessage (PG:73)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6938)
  at java.lang.reflect.Method.invoke (Native method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
"FinalizerDaemon" tid=5 Blocked 
"FinalizerDaemon" daemon prio=5 tid=5 Blocked
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14b82cb8 self=0x7e10b06200
  | sysTid=9770 nice=4 cgrp=default sched=0/0 handle=0x7df875e4f0
  | state=S schedstat=( 13742912 5164323 29 ) utm=1 stm=0 core=6 HZ=100
  | stack=0x7df865c000-0x7df865e000 stackSize=1037KB
  | held mutexes=
  at com.google.android.gms.ads.internal.webview.j.k (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:3)
- waiting to lock <0x0c7f7cea> (a java.lang.Object) held by thread 1
  at com.google.android.gms.ads.internal.webview.v.destroy (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:4)
- locked <0x0388bcd5> (a com.google.android.gms.ads.internal.webview.v)
  at com.google.android.gms.ads.internal.webview.t.destroy (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:10)
  at com.google.android.gms.ads.nonagon.ad.rewardedvideo.c.finalize (com.google.android.gms.policy_ads_fdr_dynamite@20300000@20300000.251657827.251657827:5)
  at java.lang.Daemons$FinalizerDaemon.doFinalize (Daemons.java:250)
  at java.lang.Daemons$FinalizerDaemon.runInternal (Daemons.java:237)
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
  at java.lang.Thread.run (Thread.java:764)
"Jit thread pool worker thread 0" tid=2 Native 
"Jit thread pool worker thread 0" daemon prio=5 tid=2 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b829b8 self=0x7e05c0e000
  | sysTid=9767 nice=9 cgrp=default sched=0/0 handle=0x7e05f914f0
  | state=S schedstat=( 59427072 63053855 180 ) utm=4 stm=1 core=6 HZ=100
  | stack=0x7e05e93000-0x7e05e95000 stackSize=1021KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 00000000000e1f30  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+152)
  #02  pc 0000000000476c50  /system/lib64/libart.so (_ZN3art10ThreadPool7GetTaskEPNS_6ThreadE+256)
  #03  pc 0000000000476204  /system/lib64/libart.so (_ZN3art16ThreadPoolWorker3RunEv+124)
  #04  pc 0000000000475cd0  /system/lib64/libart.so (_ZN3art16ThreadPoolWorker8CallbackEPv+144)
  #05  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #06  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"ReferenceQueueDaemon" tid=4 Waiting 
"ReferenceQueueDaemon" daemon prio=5 tid=4 Waiting
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14b82ac8 self=0x7e10b05800
  | sysTid=9769 nice=4 cgrp=default sched=0/0 handle=0x7df88634f0
  | state=S schedstat=( 11380882 2243595 19 ) utm=1 stm=0 core=0 HZ=100
  | stack=0x7df8761000-0x7df8763000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x0153abdb> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
  at java.lang.Daemons$ReferenceQueueDaemon.runInternal (Daemons.java:178)
- locked <0x0153abdb> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
  at java.lang.Thread.run (Thread.java:764)
"FinalizerWatchdogDaemon" tid=7 Sleeping 
"FinalizerWatchdogDaemon" daemon prio=5 tid=7 Sleeping
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14b82fe8 self=0x7e10b06c00
  | sysTid=9771 nice=4 cgrp=default sched=0/0 handle=0x7df86594f0
  | state=S schedstat=( 584324 1552082 10 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7df8557000-0x7df8559000 stackSize=1037KB
  | held mutexes=
  at java.lang.Thread.sleep (Native method)
- sleeping on <0x055f1678> (a java.lang.Object)
  at java.lang.Thread.sleep (Thread.java:373)
- locked <0x055f1678> (a java.lang.Object)
  at java.lang.Thread.sleep (Thread.java:314)
  at java.lang.Daemons$FinalizerWatchdogDaemon.sleepFor (Daemons.java:342)
  at java.lang.Daemons$FinalizerWatchdogDaemon.waitForFinalization (Daemons.java:364)
  at java.lang.Daemons$FinalizerWatchdogDaemon.runInternal (Daemons.java:281)
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
  at java.lang.Thread.run (Thread.java:764)
"Binder:9762_1" tid=8 Native 
"Binder:9762_1" prio=5 tid=8 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b83138 self=0x7e05c1b400
  | sysTid=9773 nice=0 cgrp=default sched=0/0 handle=0x7df83514f0
  | state=S schedstat=( 9719484 10947339 127 ) utm=0 stm=0 core=1 HZ=100
  | stack=0x7df8257000-0x7df8259000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000006acb0  /system/lib64/libc.so (__ioctl+4)
  #01  pc 0000000000024f78  /system/lib64/libc.so (ioctl+132)
  #02  pc 0000000000061f24  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+256)
  #03  pc 0000000000062094  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+24)
  #04  pc 0000000000062764  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+60)
  #05  pc 00000000000830d4  /system/lib64/libbinder.so (???)
  #06  pc 000000000001160c  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+280)
  #07  pc 00000000000fcda4  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+136)
  #08  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #09  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"Binder:9762_2" tid=9 Native 
"Binder:9762_2" prio=5 tid=9 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b83210 self=0x7e10b86600
  | sysTid=9774 nice=0 cgrp=default sched=0/0 handle=0x7df82544f0
  | state=S schedstat=( 10041146 8707241 129 ) utm=1 stm=0 core=1 HZ=100
  | stack=0x7df815a000-0x7df815c000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000006acb0  /system/lib64/libc.so (__ioctl+4)
  #01  pc 0000000000024f78  /system/lib64/libc.so (ioctl+132)
  #02  pc 0000000000061f24  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+256)
  #03  pc 0000000000062094  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+24)
  #04  pc 0000000000062764  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+60)
  #05  pc 00000000000830d4  /system/lib64/libbinder.so (???)
  #06  pc 000000000001160c  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+280)
  #07  pc 00000000000fcda4  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+136)
  #08  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #09  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"Profile Saver" tid=10 Native 
"Profile Saver" daemon prio=5 tid=10 Native
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14b832e8 self=0x7e05c26800
  | sysTid=9784 nice=9 cgrp=default sched=0/0 handle=0x7df719c4f0
  | state=S schedstat=( 23652445 6249166 31 ) utm=2 stm=0 core=5 HZ=100
  | stack=0x7df70a2000-0x7df70a4000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 00000000000e1f30  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+152)
  #02  pc 0000000000309a8c  /system/lib64/libart.so (_ZN3art12ProfileSaver3RunEv+352)
  #03  pc 000000000030b9f8  /system/lib64/libart.so (_ZN3art12ProfileSaver21RunProfileSaverThreadEPv+88)
  #04  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #05  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"Measurement Worker" tid=12 TimedWaiting 
"Measurement Worker" prio=5 tid=12 TimedWaiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b83c98 self=0x7e001f8800
  | sysTid=9794 nice=10 cgrp=default sched=0/0 handle=0x7df56444f0
  | state=S schedstat=( 55497043 39574888 195 ) utm=5 stm=0 core=4 HZ=100
  | stack=0x7df5542000-0x7df5544000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x0b255151> (a java.lang.Object)
  at java.lang.Object.wait (Object.java:422)
  at il.run (com.google.android.gms.dynamite_measurementdynamite@17785048@17.7.85 (040400-253824076):15)
- locked <0x0b255151> (a java.lang.Object)
"queued-work-looper" tid=13 Native 
"queued-work-looper" prio=5 tid=13 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b83a48 self=0x7e001f7e00
  | sysTid=9792 nice=-2 cgrp=default sched=0/0 handle=0x7df3f0c4f0
  | state=S schedstat=( 49595458 8574219 119 ) utm=4 stm=0 core=5 HZ=100
  | stack=0x7df3e0a000-0x7df3e0c000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"UnityMain" tid=14 Native 
"UnityMain" prio=5 tid=14 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b80060 self=0x7e001f4200
  | sysTid=9793 nice=0 cgrp=default sched=0/0 handle=0x7df37944f0
  | state=S schedstat=( 25152717655 2907059620 29966 ) utm=1958 stm=557 core=4 HZ=100
  | stack=0x7df3692000-0x7df3694000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 0000000000067c04  /system/lib64/libc.so (pthread_cond_wait+96)
  #02  pc 00000000002d6c44  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #03  pc 000000000048a770  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #04  pc 000000000048a8c8  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #05  pc 000000000048a9d4  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #06  pc 00000000003ad99c  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #07  pc 00000000003b4c1c  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #08  pc 00000000003b4c50  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #09  pc 00000000003b4e20  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #10  pc 00000000002caa7c  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #11  pc 00000000002c50fc  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/lib/arm64/libunity.so (???)
  #12  pc 000000000010bbac  /data/app/com.livezen.retroshooting.free-ZDlWgI85wGWQfujllcGfSA==/oat/arm64/base.odex (Java_com_unity3d_player_UnityPlayer_nativeRender__+124)
  at com.unity3d.player.UnityPlayer.nativeRender (Native method)
  at com.unity3d.player.UnityPlayer.c (unavailable)
  at com.unity3d.player.UnityPlayer$e$1.handleMessage (unavailable)
  at android.os.Handler.dispatchMessage (Handler.java:101)
  at android.os.Looper.loop (Looper.java:164)
  at com.unity3d.player.UnityPlayer$e.run (unavailable)
"Binder:9762_3" tid=15 Native 
"Binder:9762_3" prio=5 tid=15 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b83f80 self=0x7e0010f800
  | sysTid=9795 nice=0 cgrp=default sched=0/0 handle=0x7df39944f0
  | state=S schedstat=( 9574323 19599686 101 ) utm=0 stm=0 core=2 HZ=100
  | stack=0x7df389a000-0x7df389c000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000006acb0  /system/lib64/libc.so (__ioctl+4)
  #01  pc 0000000000024f78  /system/lib64/libc.so (ioctl+132)
  #02  pc 0000000000061f24  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+256)
  #03  pc 0000000000062094  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+24)
  #04  pc 0000000000062764  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+60)
  #05  pc 00000000000830d4  /system/lib64/libbinder.so (???)
  #06  pc 000000000001160c  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+280)
  #07  pc 00000000000fcda4  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+136)
  #08  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #09  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"UnityChoreographer" tid=16 Native 
"UnityChoreographer" prio=5 tid=16 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b84008 self=0x7de44e3c00
  | sysTid=9830 nice=-2 cgrp=default sched=0/0 handle=0x7de31ff4f0
  | state=S schedstat=( 1105112370 160092197 3210 ) utm=70 stm=40 core=2 HZ=100
  | stack=0x7de30fd000-0x7de30ff000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 00000000000e1f30  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+152)
  #02  pc 0000000000319c28  /system/lib64/libart.so (_ZN3art3JNI16CallObjectMethodEP7_JNIEnvP8_jobjectP10_jmethodIDz+560)
  #03  pc 00000000001336a4  /system/lib64/libandroid_runtime.so (_ZN7android26NativeDisplayEventReceiver13dispatchVsyncElij+48)
  #04  pc 0000000000040d28  /system/lib64/libandroidfw.so (_ZN7android22DisplayEventDispatcher11handleEventEiiPv+164)
  #05  pc 0000000000015eb4  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+808)
  #06  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #07  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #08  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"OkHttp ConnectionPool" tid=18 TimedWaiting 
"OkHttp ConnectionPool" daemon prio=4 tid=18 TimedWaiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14b840f0 self=0x7dfff93200
  | sysTid=9832 nice=10 cgrp=default sched=0/0 handle=0x7de30fa4f0
  | state=S schedstat=( 17135057 12734579 49 ) utm=1 stm=0 core=0 HZ=100
  | stack=0x7de2ff8000-0x7de2ffa000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x03507db6> (a com.android.okhttp.ConnectionPool)
  at com.android.okhttp.ConnectionPool$1.run (ConnectionPool.java:101)
- locked <0x03507db6> (a com.android.okhttp.ConnectionPool)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1162)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
  at java.lang.Thread.run (Thread.java:764)
"Chrome_ProcessLauncherThread" tid=21 Native 
"Chrome_ProcessLauncherThread" prio=5 tid=21 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e45848 self=0x7decae9a00
  | sysTid=9868 nice=0 cgrp=default sched=0/0 handle=0x7dd11164f0
  | state=S schedstat=( 7241404 3822397 36 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7dd1014000-0x7dd1016000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"GoogleApiHandler" tid=25 Native 
"GoogleApiHandler" prio=5 tid=25 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e46338 self=0x7de1d57e00
  | sysTid=9877 nice=9 cgrp=default sched=0/0 handle=0x7dd0bff4f0
  | state=S schedstat=( 6822920 5746769 39 ) utm=0 stm=0 core=3 HZ=100
  | stack=0x7dd0afd000-0x7dd0aff000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"Chrome_IOThread" tid=26 Native 
"Chrome_IOThread" prio=7 tid=26 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e46440 self=0x7de4532400
  | sysTid=9886 nice=-4 cgrp=default sched=0/0 handle=0x7dd01b34f0
  | state=S schedstat=( 2193225410 1038047934 10215 ) utm=122 stm=97 core=4 HZ=100
  | stack=0x7dd00b9000-0x7dd00bb000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000001e3b688  /data/app/com.android.chrome-JIeAVYvUOoXcL0OO-m9n2g==/base.apk (???)
"NetworkService" tid=27 Native 
"NetworkService" prio=5 tid=27 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e464c8 self=0x7deca3c400
  | sysTid=9890 nice=0 cgrp=default sched=0/0 handle=0x7dcfebc4f0
  | state=S schedstat=( 815515989 399798147 1909 ) utm=46 stm=35 core=4 HZ=100
  | stack=0x7dcfdc2000-0x7dcfdc4000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000001e3b688  /data/app/com.android.chrome-JIeAVYvUOoXcL0OO-m9n2g==/base.apk (???)
"PlatformServiceBridgeHandlerThread" tid=28 Native 
"PlatformServiceBridgeHandlerThread" prio=5 tid=28 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e46550 self=0x7dd0d08c00
  | sysTid=9902 nice=0 cgrp=default sched=0/0 handle=0x7dcf9cb4f0
  | state=S schedstat=( 7359014 5941302 44 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7dcf8c9000-0x7dcf8cb000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"ThreadPoolForeg" tid=29 Native 
"ThreadPoolForeg" prio=5 tid=29 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e468f8 self=0x7dd0d35800
  | sysTid=9885 nice=0 cgrp=default sched=0/0 handle=0x7dd02b04f0
  | state=S schedstat=( 207063796 114657859 571 ) utm=7 stm=13 core=5 HZ=100
  | stack=0x7dd01b6000-0x7dd01b8000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddf0  /system/lib64/libc.so (syscall+32)
  #01  pc 0000000000067ce8  /system/lib64/libc.so (pthread_cond_timedwait+156)
  #02  pc 0000000001ccbea0  /data/app/com.android.chrome-JIeAVYvUOoXcL0OO-m9n2g==/base.apk (???)
"GAC_Executor[0]" tid=30 Waiting 
"GAC_Executor[0]" prio=5 tid=30 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e46a68 self=0x7dd0d8cc00
  | sysTid=9908 nice=0 cgrp=default sched=0/0 handle=0x7dcf7c94f0
  | state=S schedstat=( 2211769 1957030 12 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7dcf6c7000-0x7dcf6c9000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x00572ab7> (a java.lang.Object)
  at java.lang.Thread.parkFor$ (Thread.java:2135)
- locked <0x00572ab7> (a java.lang.Object)
  at sun.misc.Unsafe.park (Unsafe.java:358)
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
  at NO.run (PG:5)
  at java.lang.Thread.run (Thread.java:764)
"CleanupReference" tid=31 Waiting 
"CleanupReference" daemon prio=5 tid=31 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e47030 self=0x7dd0d13a00
  | sysTid=9909 nice=0 cgrp=default sched=0/0 handle=0x7dcf6c44f0
  | state=S schedstat=( 200885 140990 3 ) utm=0 stm=0 core=2 HZ=100
  | stack=0x7dcf5c2000-0x7dcf5c4000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x04ab2224> (a java.lang.Object)
  at java.lang.Object.wait (Object.java:422)
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:188)
- locked <0x04ab2224> (a java.lang.Object)
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:209)
  at aKd.run (PG:3)
"AudioThread" tid=32 Native 
"AudioThread" prio=5 tid=32 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e476c0 self=0x7de1fbba00
  | sysTid=9897 nice=0 cgrp=default sched=0/0 handle=0x7dcfdbf4f0
  | state=S schedstat=( 4498807 2883230 33 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7dcfcc5000-0x7dcfcc7000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 0000000000067c04  /system/lib64/libc.so (pthread_cond_wait+96)
  #02  pc 0000000001ed0fe4  /data/app/com.android.chrome-JIeAVYvUOoXcL0OO-m9n2g==/base.apk (???)
"Chrome_InProcGpuThread" tid=33 Native 
"Chrome_InProcGpuThread" prio=7 tid=33 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e47830 self=0x7dcf213400
  | sysTid=9915 nice=-4 cgrp=default sched=0/0 handle=0x7dcf1ff4f0
  | state=S schedstat=( 1046020252 185669412 1299 ) utm=75 stm=29 core=2 HZ=100
  | stack=0x7dcf105000-0x7dcf107000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddf0  /system/lib64/libc.so (syscall+32)
  #01  pc 0000000000067ce8  /system/lib64/libc.so (pthread_cond_timedwait+156)
  #02  pc 0000000001ccbea0  /data/app/com.android.chrome-JIeAVYvUOoXcL0OO-m9n2g==/base.apk (???)
"GoogleApiHandler" tid=34 Native 
"GoogleApiHandler" prio=5 tid=34 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e479a0 self=0x7dcf20da00
  | sysTid=9921 nice=9 cgrp=default sched=0/0 handle=0x7dcebff4f0
  | state=S schedstat=( 17847288 14419067 76 ) utm=1 stm=0 core=0 HZ=100
  | stack=0x7dceafd000-0x7dceaff000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"GAC_Executor[1]" tid=35 Waiting 
"GAC_Executor[1]" prio=5 tid=35 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e480c8 self=0x7dd0d8d600
  | sysTid=9923 nice=0 cgrp=default sched=0/0 handle=0x7dceafa4f0
  | state=S schedstat=( 2180107 11458906 19 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7dce9f8000-0x7dce9fa000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait (Native method)
- waiting on <0x0d1cb18d> (a java.lang.Object)
  at java.lang.Thread.parkFor$ (Thread.java:2135)
- locked <0x0d1cb18d> (a java.lang.Object)
  at sun.misc.Unsafe.park (Unsafe.java:358)
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
  at NO.run (PG:5)
  at java.lang.Thread.run (Thread.java:764)
"Binder:9762_4" tid=36 Native 
"Binder:9762_4" prio=5 tid=36 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e48a00 self=0x7dcf20e400
  | sysTid=9924 nice=0 cgrp=default sched=0/0 handle=0x7dce9f54f0
  | state=S schedstat=( 8643756 17277915 139 ) utm=0 stm=0 core=1 HZ=100
  | stack=0x7dce8fb000-0x7dce8fd000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 00000000000e1f30  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+152)
  #02  pc 000000000031b8bc  /system/lib64/libart.so (_ZN3art3JNI18CallBooleanMethodVEP7_JNIEnvP8_jobjectP10_jmethodIDSt9__va_list+492)
  #03  pc 0000000000101e48  /system/lib64/libandroid_runtime.so (???)
  #04  pc 00000000001680d8  /system/lib64/libandroid_runtime.so (???)
  #05  pc 0000000000055be4  /system/lib64/libbinder.so (_ZN7android7BBinder8transactEjRKNS_6ParcelEPS1_j+136)
  #06  pc 00000000000623e0  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14executeCommandEi+536)
  #07  pc 0000000000062118  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+156)
  #08  pc 0000000000062764  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+60)
  #09  pc 00000000000830d4  /system/lib64/libbinder.so (???)
  #10  pc 000000000001160c  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+280)
  #11  pc 00000000000fcda4  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+136)
  #12  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #13  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"LooperProvider" tid=37 Native 
"LooperProvider" prio=5 tid=37 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e48d60 self=0x7dcf20ee00
  | sysTid=9926 nice=0 cgrp=default sched=0/0 handle=0x7dce8f84f0
  | state=S schedstat=( 8583329 1108227 43 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7dce7f6000-0x7dce7f8000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000006abc4  /system/lib64/libc.so (__epoll_pwait+8)
  #01  pc 00000000000205a8  /system/lib64/libc.so (epoll_pwait+48)
  #02  pc 0000000000015c1c  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+144)
  #03  pc 0000000000015b04  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+108)
  #04  pc 000000000015ae18  /system/lib64/libandroid_runtime.so (???)
  #05  pc 00000000001ec1cc  /system/framework/arm64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+140)
  at android.os.MessageQueue.nativePollOnce (Native method)
  at android.os.MessageQueue.next (MessageQueue.java:325)
  at android.os.Looper.loop (Looper.java:142)
  at android.os.HandlerThread.run (HandlerThread.java:65)
"Binder:9762_5" tid=44 Native 
"Binder:9762_5" prio=5 tid=44 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x13e4a378 self=0x7dcec12a00
  | sysTid=9944 nice=0 cgrp=default sched=0/0 handle=0x7dcdfcb4f0
  | state=S schedstat=( 21995887 41289172 184 ) utm=2 stm=0 core=1 HZ=100
  | stack=0x7dcded1000-0x7dcded3000 stackSize=1005KB
  | held mutexes=
  #00  pc 000000000001ddec  /system/lib64/libc.so (syscall+28)
  #01  pc 00000000000e1f30  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+152)
  #02  pc 000000000031b8bc  /system/lib64/libart.so (_ZN3art3JNI18CallBooleanMethodVEP7_JNIEnvP8_jobjectP10_jmethodIDSt9__va_list+492)
  #03  pc 0000000000101e48  /system/lib64/libandroid_runtime.so (???)
  #04  pc 00000000001680d8  /system/lib64/libandroid_runtime.so (???)
  #05  pc 0000000000055be4  /system/lib64/libbinder.so (_ZN7android7BBinder8transactEjRKNS_6ParcelEPS1_j+136)
  #06  pc 00000000000623e0  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14executeCommandEi+536)
  #07  pc 0000000000062118  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+156)
  #08  pc 0000000000062764  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+60)
  #09  pc 00000000000830d4  /system/lib64/libbinder.so (???)
  #10  pc 000000000001160c  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+280)
  #11  pc 00000000000fcda4  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+136)
  #12  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #13  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"Signal Catcher" tid=3 Runnable 
"Signal Catcher" daemon prio=5 tid=3 Runnable
  | group="system" sCount=0 dsCount=0 flags=0 obj=0x14b82a40 self=0x7e10ac5400
  | sysTid=9768 nice=0 cgrp=default sched=0/0 handle=0x7e05bff4f0
  | state=R schedstat=( 30716979 8081460 28 ) utm=2 stm=1 core=0 HZ=100
  | stack=0x7e05b05000-0x7e05b07000 stackSize=1005KB
  | held mutexes= "mutator lock"(shared held)
  #00  pc 0000000000397a18  /system/lib64/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_9ArtMethodEPv+212)
  #01  pc 000000000045e3c8  /system/lib64/libart.so (_ZNK3art6Thread9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMapb+348)
  #02  pc 00000000004753a4  /system/lib64/libart.so (_ZN3art14DumpCheckpoint3RunEPNS_6ThreadE+880)
  #03  pc 000000000046de58  /system/lib64/libart.so (_ZN3art10ThreadList13RunCheckpointEPNS_7ClosureES2_+476)
  #04  pc 000000000046d868  /system/lib64/libart.so (_ZN3art10ThreadList4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEb+796)
  #05  pc 000000000046d408  /system/lib64/libart.so (_ZN3art10ThreadList14DumpForSigQuitERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+920)
  #06  pc 0000000000442930  /system/lib64/libart.so (_ZN3art7Runtime14DumpForSigQuitERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+196)
  #07  pc 000000000044c268  /system/lib64/libart.so (_ZN3art13SignalCatcher13HandleSigQuitEv+1676)
  #08  pc 000000000044b0a0  /system/lib64/libart.so (_ZN3art13SignalCatcher3RunEPv+396)
  #09  pc 0000000000068560  /system/lib64/libc.so (_ZL15__pthread_startPv+36)
  #10  pc 000000000001f9f4  /system/lib64/libc.so (__start_thread+68)
"HeapTaskDaemon" tid=6 Blocked 
"HeapTaskDaemon" daemon prio=5 tid=6 Blocked
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14b84250 self=0x7e10b07600
  | sysTid=9772 nice=4 cgrp=default sched=0/0 handle=0x7df85544f0
  | state=S schedstat=( 350645591 71982454 318 ) utm=30 stm=5 core=4 HZ=100
  | stack=0x7df8452000-0x7df8454000 stackSize=1037KB
  | held mutexes=
  #00  pc 000000000001ddf0  /system/lib64/libc.so (syscall+32)
  #01  pc 00000000000e2358  /system/lib64/libart.so (_ZN3art17ConditionVariable9TimedWaitEPNS_6ThreadEli+172)
  #02  pc 0000000000240f84  /system/lib64/libart.so (_ZN3art2gc13TaskProcessor7GetTaskEPNS_6ThreadE+320)
  #03  pc 0000000000241864  /system/lib64/libart.so (_ZN3art2gc13TaskProcessor11RunAllTasksEPNS_6ThreadE+92)
  #04  pc 000000000020c8bc  /system/framework/arm64/boot-core-libart.oat (Java_dalvik_system_VMRuntime_runHeapTasks__+124)
  at dalvik.system.VMRuntime.runHeapTasks (Native method)
- waiting to lock an unknown object
  at java.lang.Daemons$HeapTaskDaemon.runInternal (Daemons.java:461)
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
  at java.lang.Thread.run (Thread.java:764)

**Comment by stowy** [comment]
Please provide information on how to reproduce this issue and the devices it has occurred on.

**Comment by livezencorp** [comment]
fortunave3g, Android 5.0
addison, Android 8.0
albus, Android 8.0
G8341, Android 8.0

**Comment by stowy** [comment]
This is a duplicate of #947 - the fix is pending release by the Android SDK team.
